README.md

```diff
@@ -1 +1,24 @@
-# Beyond-07-team-4
\ No newline at end of file
+# Beyond-07-team-4
+
+**Reqirements:**
+
+Adding a Vagrantfile for spinning up a development VM.
+
+Vagrant is a tool for using virtual machines to share development environments.
+Learn more about it at:
+https://www.vagrantup.com/
+
+Please make sure you have Vagrant installed on your computer, if not then you can download from here:
+https://www.vagrantup.com/downloads
+
+Also, make sure that you have VirtualBox installed. VirtualBox is a hypervisor, which lets us create and run virtual machines.
+The page below include links for downloading VirtualBox for various OS:
+https://www.virtualbox.org/wiki/Downloads
+
+In order for it all to work, please enable virtualization hardware on your computer. An explanation on how to do it can be found here:
+https://www.virtualmetric.com/blog/how-to-enable-hardware-virtualization
+
+Then, when you have all those you can bring up the VM by using the command: vagrant up.
```

Comment:
If there is not correlation to vagrantfile, i think vagrant init is a must to create the initial vagrant file in your directory, isnt it?

Need to check if vagrantfile not created to use vagrant init
In case there is vagrantfile in directory use the vagrant up

Reply:
Added to the updated README.md